milestone-3/frontend/src/components/about.jsx

```diff
@@ -0,0 +1,28 @@
+import './about.css';
+import Adam from "./IMG_2737.jpg";
+import Navbar from './navbar';
+
+
+export default function About() {
+    return (
+    //boring about page that never changes, route to path "/about"
+    <header>  
+        <div class="title">
+            <h1>Who is Adam Meza???!</h1>
+        </div>
+
+        <Navbar />
```

Comment:
On most pages we'd recommend putting the `<Navbar />` in App.js since its rendered on every page. However, your case is unique since you render the title of the page above the navbar. If you did want to move `<Navbar />` into App.js, you would need to also need to move the title into app.js and change the title by storing the title state in App.js and passing `setTitle` to all of the different page components. This is a bit of extra work though, so don't feel like you have to!